src/framework/theme/service/style/styleConsumer.service.ts

```diff
@@ -14,6 +14,7 @@ interface ComponentStyleMetaType {
   appearance: string;
   variants: string[];
   states: string[];
+  params: string[];
```

Comment:
Don't think this should be here

Reply:
Why not?

Reply:
This model describes derived style, not one of we got from generated. So here we don't know anything about valid parameters